test/views/duty_deferment_account/DutyDefermentAccountSpec.scala

```diff
@@ -60,6 +63,48 @@ class DutyDefermentAccountSpec extends SpecBase {
     }
   }
 
+  "DutyDefermentAccount" should {
```

Comment:
These new tests are for model (method dropOldMonths) but not for the view. Could you please these tests in a new test class named (DutyDefermentAccountSpec in viewmodels package) and write the tests for the views (for the code changes we have made in the view). If you find it difficult to write the tests for the view, add a note in the new ticket (to move the complex logic out of the view).